scenario/LiquidationScenario.ts

```diff
@@ -161,6 +161,37 @@ scenario(
   }
 );
 
+scenario(
+  'Comet#liquidation > user can end up with a minted supply',
+  {
+    tokenBalances: {
+      $comet: { $base: 1000 },
+    },
+    cometBalances: {
+      albert: {
+        $base: -1000,
+        $asset0: 0.001
+      },
+    },
+  },
+  async ({ comet, actors }, context, world) => {
+    const { albert, betty } = actors;
+
+    await world.increaseTime(
+      await timeUntilUnderwater({
+        comet,
+        actor: albert,
+      })
+    );
+
+    const ab0 = await betty.absorb({ absorber: betty.address, accounts: [albert.address] });
+    expect(ab0.events[2].event).to.be.equal('Transfer');
```

Comment:
looks like `ab0.events` can be null.

you could update this to: `ab0.events?.[2]?.event` if you wanted the expectations to fall when there were no events (instead of failing with a JS error about not finding `event` on `undefined`)

Reply:
This is the test we want ;) See the note at the top

Reply:
Good call 🦅